### PR TITLE
Add links to extensions

### PR DESCRIPTION
**Pull request by szarnyasg**

### CODE REVIEW

File: docs/extensions/overview.md [hunk 1]
```diff
@@ -30,24 +30,24 @@ select * from duckdb_extensions();
...
+| inet                                                               | Adds support for IP-related data types and functions                 |                 |
+| jemalloc                                                           | Overwrites system allocator with JEMalloc                            |                 |
+| json                                                               | Adds support for JSON operations                                     |                 |
+| parquet                                                            | Adds support for reading and writing parquet files                   |                 |
+| [postgres_scanner](https://github.com/duckdblabs/postgres_scanner) | Adds support for reading from a Postgres database                    | postgres        |
+| [spatial](https://github.com/duckdblabs/duckdb_spatial)            | Adds support for geospatial data processing                          |                 |
+| [sqlite_scanner](https://github.com/duckdblabs/sqlite_scanner)     | Adds support for reading SQLite database files                       | sqlite, sqlite3 |
```
Comment (Mause) [inline docs/extensions/overview.md:47]: Maybe these should be links to the docs instead? And we can add octocats (:octocat:) to link to the repos, as they'll be less often used

Comment (szarnyasg) [reply docs/extensions/overview.md]: Good point. I linked to the docs now except for Arrow where we currently only have guides.

File: docs/extensions/overview.md [hunk 2]
```diff
@@ -30,24 +30,25 @@ select * from duckdb_extensions();
...
-| spatial           | Adds support for geospatial data processing                          |                 |
-| sqlite_scanner    | Adds support for reading SQLite database files                       | sqlite, sqlite3 |
-| substrait        | Adds support for the Substrait integration                           |
-| tpcds             | Adds TPC-DS data generation and query support                        |                 |
-| tpch              | Adds TPC-H data generation and query support                         |                 |
-| visualizer        |                                                                      |                 |
+| Extension name                         | Description                                                          | Aliases         |
+| -------------------------------------- | -------------------------------------------------------------------- | --------------- |
+| autocomplete                           | Adds supports for autocomplete in the shell                          |                 |
+| [arrow](../guides/python/sql_on_arrow) | A zero-copy data integration between Apache Arrow and DuckDB         |                 |
```
Comment (Mause) [inline docs/extensions/overview.md:36]: This doesn't actually cover this extension, which I know is confusing (Arrow is kinda an overarching project name that covers several related technologies). I don't think we have any docs on the extension besides what is in the repo

**Review by Mause — CHANGES_REQUESTED** [review]

**Comment by szarnyasg** [comment]
I added the links to the repositories with the Octocat icon (from an SVG image that I resized with Inkscape). Preview:

Screenshot 2023-08-01 at 10 17 17

I had to edit the underlying CSS - I'm not particularly happy with the `margin-bottom: -4px` property but I think we can leave it as is now.

File: css/main.scss [hunk 3]
```diff
@@ -1032,6 +1032,12 @@ body.documentation{
 				margin-bottom: 50px;
 				margin-top: 50px;
 			}
+			table{ img{
```
Comment (Mause) [inline css/main.scss:1035]: This definitely needs tweaking later, but will do for now

**Review by Mause — APPROVED** [review]

**Pull request merged**